exercism-modern.el

```diff
@@ -300,6 +321,7 @@ Pass prefix BUFFER-PREFIX-ARG to prompt for a buffer instead."
                                                        :file ,difficulty-svg))
                                                     (propertize blurb 'face text-face))))))
           tabulated-list-padding 4)
+    (message "") ;; clear any previous messages
```

Comment:
Not sure this is needed

Reply:
Else the "downloading X track ..." will continue to stay in the minibuffer.

Reply:
Oh, put that in the downloading method then